### DISCUSSION TITLE
Add more "Known Files"

### DISCUSSION THREAD

**Discussion by ghost** [biomejs/biome#840] [Ideas]
Currently, Biome only ignores these known files `package.json`,`package-lock.json`,`tsconfig.json`, and`jsconfig.json`. I think `bun.lockb`, `package-lock.yaml`, and yarn should be included in this list.

**Comment by Conaclos** [comment]
Good idea! PR are welcomed :)

**Comment by ematipico** [comment]
@jacobthesheep do you want to send a PR for the bun file? Here's an example: https://github.com/biomejs/biome/pull/875

**Comment by xnuk** [comment]
But how do I enable for this? I'm bit surprised that Biome does not formats `deno.json` and `package.json`. As a person doesn't like 2-space indentation and `deno fmt`(`dprint`), I want to use Biome instead for these files.

**Reply by ematipico** [reply]
I believe we are going to remove them soon
